components/Snackbar/src/MDCSnackbarManager.h

```diff
@@ -249,10 +256,11 @@
  If disabled, the font size will only be adjusted if a scaled font has been provided.
  This behavior most closely matches UIKit's.
 
- Default value is YES, but this flag will eventually default to NO and then be deprecated
- and deleted.
+ Default value is NO.
  */
-@property(nonatomic, assign) BOOL adjustsFontForContentSizeCategoryWhenScaledFontIsUnavailable;
+@property(nonatomic, assign)
+    BOOL adjustsFontForContentSizeCategoryWhenScaledFontIsUnavailable __deprecated_msg(
+        "Please provide a scaled font and set adjustsFontForContentSizeCategory to YES.");
```

Comment:
Tip: Nit: Suggestion: "Use UIFontMetrics and UIContentSizeCategoryAdjusting on iOS 11+ or MDCFontScaler on earlier versions."